.github/workflows/ci.yaml

```diff
@@ -47,6 +47,22 @@ jobs:
       - name: Run tests
         run: npm test
 
+  esm_cjs:
+    name: Types & Entrypoints
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+          cache: npm
+      - name: Install dependencies
+        run: npm ci
+      - name: Verify types & entrypoints
+        run: npx --yes @arethetypeswrong/cli --pack .
```

Comment:
This step is just another test, so it could get shoved into the `test` matrix and we could skip the extra `npm ci`. But then we'd need a conditional skip for jus that step to only run it once, instead of for everything in the matrix. 🤔

Reply:
I had considered that but it's clearer what the issue is if this separate job fails + it doesn't need to be checked for everything in the matrix.